docs/user-guide/102-policy-syntax.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Policy Syntax"
+tag: "user-guide"
+---
+
+The policy contains the following fields:
+- `capture`: Contains some simple expressions
+to evaluate the current network configuration and store the result at 
+variables.
+- `desiredState`: Contains network state yaml with references to `capture` entries.
+
+__Example:__
+```yaml
+{% include_absolute 'examples/bridge-on-default-gw-dhcp/policy.md' %}
+```
+
+## Capture syntax
+
+The capture is a map of capture entries with an identifier as the key that can 
+be referenced with ```capture.[id]```
+
+The map value is a capture entry expression that evaluates the current 
+network state and stores the result.
+
+The following is a semi-formal definition of the capture entry expression:
+
+```
+<letter> ::= "A" | "B" | "C" | "D" | "E" | "F" | "G"
+       | "H" | "I" | "J" | "K" | "L" | "M" | "N"
+       | "O" | "P" | "Q" | "R" | "S" | "T" | "U"
+       | "V" | "W" | "X" | "Y" | "Z" | "a" | "b"
+       | "c" | "d" | "e" | "f" | "g" | "h" | "i"
+       | "j" | "k" | "l" | "m" | "n" | "o" | "p"
+       | "q" | "r" | "s" | "t" | "u" | "v" | "w"
+       | "x" | "y" | "z"
+<digit> ::= [0-9]
+<number> ::= <digit>+
+<identity> ::= <letter> ( <digit> | "-" | <letter> )*
+<dot> ::= "."
+<path> ::= <identity> ( <dot> <identity> )*
```

Comment:
Continuing https://github.com/nmstate/nmpolicy/pull/80#discussion_r790479392

`<path> ::= <identifier> ( <dot> (<identity> | <number> )+)*`
why the `+` is needed?

Reply:
Si we don't end with something like `foo.` it forces to have at least one occurency

Reply:
So the `+` should be instead the `*`. Isn't it?

Reply:
Path with just identity is valid

Reply:
In this case - `<path> ::= <identifier> ( <dot> (<identity> | <number> ))*` seems enough.